src/layout/navbar/search/index.tsx

```
@@ -63,92 +84,110 @@ const SearchInput: React.FC = () => {
+          >
+            {item => (
+              <ListboxItem
+                key={item.key}
+                onPress={() => {
+                  setOpen(false);
+                  setValue(item.value);
+                  submitSearch(item.value);
+                }}
+              >
+                <span dangerouslySetInnerHTML={{ __html: item.name }} />
+              </ListboxItem>
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)